### PR TITLE
Feature/user variables

### PR DESCRIPTION
**Pull request by jonathanmendoza-tx**
# Description

added ability for user to select and change variables: jitter, depth, and truncation

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

deployed on AWS EB and successfully created videos using new variables

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: helper.py [hunk 1]
```diff
@@ -37,15 +39,18 @@ def check_entry(preview, video_id, resolution, im_group):
...
+def generate_and_save(preview, video_id, resolution, classes, jitter, 
+						depth, truncation):
 
 	song = requests.get(preview)
 
 	open(f"{video_id}.mp3", 'wb').write(song.content)
 
-	noise_vectors, class_vectors = song_analysis(f"{video_id}.mp3", classes)
+	noise_vectors, class_vectors = song_analysis(f"{video_id}.mp3", classes, 
+												jitter, depth, truncation)
```
Comment (StevenMElliott) [inline helper.py:50]: Why is jitter/depth so far out to the right?

File: visualize.py [hunk 2]
```diff
@@ -70,10 +70,11 @@ def sensitivity_tempo(tempo_sensitivity=0.25):
 
 	return tempo_sensitivity
 
+
+# truncation
```
Comment (StevenMElliott) [inline visualize.py:74]: We should get rid of these extra comments that are now dead.

File: visualize.py [hunk 3]
```diff
@@ -434,6 +435,6 @@ def save_video(frames, song, outname):
 	clip.write_videofile(outname + ".mp4", audio_codec='aac')
 
 	# saves thumbnail
-	scipy.misc.imsave(outname + ".jpg", frames[-1])
+	imsave(outname + ".jpg", frames[-1])
```
Comment (StevenMElliott) [inline visualize.py:438]: Trimmed the fat

**Review by StevenMElliott — APPROVED** [review]
Three knobs down.

**Review by adphelps — APPROVED** [review]

**Pull request merged**